backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/repos/AdministratorsRepository.kt

```diff
@@ -122,17 +122,17 @@ object AdministratorsRepository {
         administrator.notificationOnVerification = notificationSettings.notificationOnVerification
     }
 
-    fun getNotificationRecipientsForApplication(project: String): List<AdministratorEntity> =
+    fun getNotificationRecipientsForApplication(project: String, regionId: Int): List<AdministratorEntity> =
         transaction {
             (Administrators innerJoin Projects).select {
-                (Projects.project eq project) and (Administrators.notificationOnApplication eq true)
+                (Projects.project eq project) and (Administrators.notificationOnApplication eq true) and (Administrators.regionId eq regionId)
             }.let { AdministratorEntity.wrapRows(it) }.toList()
         }
 
-    fun getNotificationRecipientsForVerification(project: String): List<AdministratorEntity> =
+    fun getNotificationRecipientsForVerification(project: String, regionId: Int): List<AdministratorEntity> =
         transaction {
             (Administrators innerJoin Projects).select {
-                (Projects.project eq project) and (Administrators.notificationOnVerification eq true)
+                (Projects.project eq project) and (Administrators.notificationOnVerification eq true) and (Administrators.regionId eq regionId)
```

Comment:
Are we missing a check for "deleted" as well?

Reply:
Good point!